azure-pipelines.yml

```diff
@@ -0,0 +1,108 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+schedules:
+  - cron: "0 0 */14 * *"
+    displayName: "Bi-monthly build"
+    branches:
+      include:
+        - master
+        - benchmarking
+
+
+
+trigger:
+- master
+- benchmarking
+
+jobs:
+
+- job: CPU_Benchmarks
+
+  pool:
+    name: Default
+    demands: 
+      - GPUs -equals 0
```

Comment:
We talked about this a bit before, but what happens if we just let the Azure scheduler assign jobs? Isn't it smart enough to assign the GPU jobs appropriately without us using this demand-for-no-gpus?

Reply:
When I've tested out the scheduler, it will sometimes give the CPU-only job to one of the machines with a GPU. If we never plan on running more than two machines at a time, I guess there wouldn't be a problem then.

Reply:
OK. In that case, I'd like a demand name that is more consistent across all of the workers. If there is a `-lessthan` and `-greaterthan` comparison for the demands, then add a capablility to all the workers called `GPU_COUNT` or something whose value is an integer. Otherwise, this demand should be `NO_GPU`.

Reply:
There appears to be no support for inequality comparison.

Reply:
:cry: Thanks for checking.